### ISSUE TITLE
Audit use of term "context"

### ISSUE THREAD

**Issue by spectranaut** [w3c/aria#1928]
We should either define the word "context" when it means some specific, or, use a work that is already defined (such as accessibility child or "owned by").

For example:
- When an article is in the context of a feed, the author MAY specify values for aria-posinset and aria-setsize.
- While aria-disabled is currently supported on columnheader, in a future version the working group plans to prohibit its use on elements with role columnheader except when the element is in the context of a grid or treegrid

**Comment by spectranaut** [comment]
Discussed in F2F meeting yesterday: https://www.w3.org/2023/05/04-aria-minutes#t01

It was decided "context" should be replace with the appropriate defined words, such as "accessibility child" or "accessibility descendant".

This work is blocked on: https://github.com/w3c/aria/issues/1150#issuecomment-1536859008

**Comment by pkra** [comment]
I believe this is no longer blocked.
